src/drivers/mediatek/mt8195/afe-dai.c

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Mediatek
```

Comment:
@kuanhsuncheng please add more info about what AFE is. Is it a digital audio interface that adds I2S support? Pointers to documentation would be also good to have.